pkg/command/completion_helper.go

```diff
@@ -14,9 +14,9 @@ const (
 	compTMCTarget    = "tmc\tFor interactions with a Tanzu Mission Control endpoint"
 
 	// Completion strings for the values of the --type flag
-	compK8sContextType = "k8s\tContext for a Kubernetes cluster"
-	compTAEContextType = "tae\tContext for a Tanzu Application Engine endpoint"
-	compTMCContextType = "tmc\tContext for a Tanzu Mission Control endpoint"
+	compK8sContextType   = "k8s\tContext for a Kubernetes cluster"
+	compTanzuContextType = "tanzu\tContext for a Tanzu endpoint"
```

Comment:
As we updated in the https://github.com/vmware-tanzu/tanzu-plugin-runtime/pull/127 should we add control plane here as well?
```suggestion
	compTanzuContextType = "tanzu\tContext for a Tanzu control plane endpoint"
```

Reply:
since this is user-facing, I would rather we keep just Tanzu and get feedback instead of introducing another term.

Reply:
Sure. I will keep `Tanzu` as is without modifying it. Thanks